src/zmq_utils.cpp

```diff
@@ -110,6 +111,18 @@ static uint8_t decoder [96] = {
     0x11, 0x12, 0x13, 0x14, 0x15, 0x16, 0x17, 0x18,
     0x19, 0x1A, 0x1B, 0x1C, 0x1D, 0x1E, 0x1F, 0x20,
     0x21, 0x22, 0x23, 0x4F, 0x00, 0x50, 0x00, 0x00
+    0xFF, 0x44, 0xFF, 0x54, 0x53, 0x52, 0x48, 0xFF, 
```

Comment:
Are you sure the intention was not to change the previous declaration rather than appending? It's a 96 bytes array with a static initialization of 192 bytes now